airflow/www/templates/airflow/task_instance.html

```diff
@@ -25,7 +25,7 @@
   <hr>
   <br>
   <h4>
-    <span class="text-muted">Task Instance:</span> <span>{{ task_id }}</span>
+    <span class="text-muted">Task Instance:</span> <span title="{{ task_id }}">{{display_name}}</span>
```

Comment:
We need to set up some fallback mechanism for old tasks. Otherwise task instances (and DAG runs) created before this change would display an empty string here.

Reply:
Agree. do you think it's better to fallback in the UI, or fill the `display_name` with `task_id` during database migration?

Reply:
Backfilling `display_name` during migration would make upgrade much too slow (and for limited benefits since most of those DAGs/tasks won’t be viewed anyway). Implementing the fallback at the UI layer would be more useful. Although I think we can do this one layer down, doing the fallback in the `display_name` _property_.

Reply:
Yes using property is better. Modified accorindgly.